src/JuMPDict.jl

```diff
@@ -81,6 +81,8 @@ macro gendict(instancename,T,idxpairs,idxsets...)
         setidxrhs = :(setindex!(d.innerArray,val))
         maplhs = :(Base.map(f::Function,d::$(typename)))
         maprhs = :($(typename)(map(f,d.innerArray),d.name,d.indexsets,d.indexexprs))
+        _wraplhs = :(_wrapper(d::$(typename),inner)) # helper function that wraps array into JuMPArray of similar type
```

Comment:
How is this different from the definition below? Does it need to be defined here?